content/blog/2018/02/2018-02-27-browser-usage.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Browser Market Share on Jenkins.io"
+tags:
+- community
+- stats
+- browser
+author: olblak
+---
+
+= Browser Market Share on Jenkins.io
+
+I recently had a look to the google analytics account used for the main jenkins website. +
+It was interesting to discover which browsers are used by Jenkins visitors and how this compares with other websites. +
+I decided to analyse one year of data from January 2017 to January 2018. +
+I selected http://gs.statcounter.com/about[statcounter] as an external data source in order to compare results from jenkins.io.
+
+
+== Jenkins Visitors
+
+Let's talk about numbers: +
+During that period, 3,496,245 users across the world visited this website. +
+This is a growth of 152.69% visitors compared to the year before. +
+92.28% of them used a desktop, 6.63% were on mobile and only 1.10% on a tablet.
+
+[cols="a,a"]
+|===
+|
+|
+|
+.Jenkins Visitors' Browser Usage
+[cols="1,10,1",width="100%"]
+!===
+!#! Browser! %
+
+! 1. 
+! Chrome
+! 69.82%
+
+! 2.
+! Firefox
+! 14.75%
+
+! 3.
+! Safari 
+! 6.44%
+
+! 4.
+! IE
+! 5.84%
+
+! 5. 
+! Edge
+! 1.59%
+
+! 6.
+! Opera
+! 0.55%
+
+! 7. 
+! Android Webview
+! 0.23%
+
+! 8.
+! Safari (in-app)
+! 0.23%
+
+! 9.
+! Ya Browser
+! 0.11%
+
+! 10.
+! UC Browser
+! 0.10%
+!===
+|
+.Jenkins Visitors' Browser Usage
+image:/images/post-images/2018-02-29/jenkins_browser_2017-2018.png[width="300",align=center]
+
+We clearly see that three browsers lead this market share with a total of 91%.
+Chrome is by far the leader followed by Firefox and Safari.
+
+Obviously those numbers taken alone don't mean anything, so let's try to put them in perspective with another data source.
+
+
+=== Fun facts:
+
+* Netscape 4.8 was still used by 15 visitors over the years.
+* 6 visitors used their Nintendo DS.
+|
+|===
+
+
+[cols="a,a"]
+|=== 
+|
+== StatCounter
+Let's now compare our previous results with the same time period from http://gs.statcounter.com/about[statcounter] who argues to collect data from 2million websites.
+Statcounter allow us to display browsers stats based on 4 different categories, 'All platform', 'Desktop', 'Tablet' or 'Mobile'. +
+For this comparison, I am interested by 'All platform' and of course 'Desktop' as 92.28% of Jenkins visitors use a desktop.
+
+|
+|
+.All Platform Browser Market Share
+[cols="1,9,1",width="100%"]
+!===
+!#!Browser ! Use
+
+! 1. 
+! Chrome
+! 54.13%
+
+! 2. 
+! Safari
+! 14.50%
+
+! 3.
+! UC Browser
+! 8.25%
+
+! 4. 
+! Firefox
+! 6.15%
+
+! 5.
+! IE
+! 3.89%
+
+! 6.
+! Opera 
+! 3.85%
+
+! 7.
+! Edge
+! 1.85%
+
+! 8.
+! Others
+! 1.76%
+
+!===
+|
+=== All Platform Browser Market Share
+Excepted Chrome who still leads the ranking with "only" 54%, all the others actors are totally shuffled. +
+Safari moved to the 2nd place with 14.50% which is a growth of 225.15%. +
+'UC Browser' appeared in the top 3 with 8.25% where only 0.10% of Jenkins visitors were using UC browser. +
```

Comment:
IOW we have little market share in China. Perhaps a reference to the docs list thread on the Chinese website might be useful?

Reply:
I don't think so, 8.76% of Jenkins visitors are coming from China, 3rd country after USA and India.
My guess, it's because UC Browser is a popular browser for mobile (in china) and jenkins.io doesn't work well on mobile.